src/main/java/com/arcbees/bourseje/client/admin/dashboard/AdminDashboardView.java

```diff
@@ -100,10 +100,12 @@ public void setCurrentState(VoteState currentState) {
 
     @Override
     public void setCandidates(List<Candidate> candidatesList, Map<String, Integer> candidateResults) {
+        candidates.setInnerHTML("");
+
         for(Candidate candidate : candidatesList) {
             CandidateWidget candidateWidget = new CandidateWidget(candidate, candidateResults.get(candidate.getName()));
```

Comment:
Not sure it's useful to have this as a widget if you're only using the element. Maybe using an Element or html template would make more sense